docs/gpu-pci.md

```diff
@@ -0,0 +1,146 @@
+# GPU-PCI enabled clusters
+
+## Overview
+
+With CAPV you can create PCI enabled GPU Kubernetes clusters on vSphere.
+
+Before you begin, be aware that:
+
+- [NVIDIA GPU Operator](https://github.com/NVIDIA/gpu-operator) allows administrators of Kubernetes clusters to manage GPU nodes just like CPU nodes in the cluster.
+- [Image Builder Project](https://github.com/kubernetes-sigs/image-builder) is required to build images with VMX version 17 and UEFI boot mode.
+
+To deploy a cluster with support for GPU nodes, use the pci flavor.
+
+## An example GPU cluster
+
+Let's create a CAPV cluster with GPU in PCI mode and run a GPU-powered vector calculation.
+
+### Source the PCIDeviceID and PCIVendorID
+
+We are using Nvidia T4's, following can be set for the same.
+
+```bash
+export VSpherePCIDeviceID=7864
+export VSpherePCIVendorID=4318
+```
+
+### Generate an nvidia-gpu cluster template
+
+`make dev-manifests`
+
+`go run ./packaging/flavorgen -f vip > ${pwd}/.cluster-api/overrides/infrastructure-vsphere/v0.8.0/cluster-template.yaml`
+
+`cat ${pwd}/.cluster-api/overrides/infrastructure-vsphere/v0.8.0/cluster-template.yaml | ~/go/src/sigs.k8s.io/cluster-api/hack/tools/bin/envsubst > /tmp/vsp-cluster.yml`
+
+### Create the cluster
+
+Apply the manifest from the previous step to your management cluster to have CAPV create a
```

Comment:
```suggestion
Set the size of the GPU nodes appropriately, since the Nvidia gpu-operator requires additional CPU and memory to install the device drivers on the VMs. 

Apply the manifest from the previous step to your management cluster to have CAPV create a
```